scripts/opt/backup-loop.sh

```diff
@@ -25,6 +25,7 @@ fi
 : "${TAR_COMPRESS_METHOD:=gzip}"  # bzip2 gzip zstd
 : "${ZSTD_PARAMETERS:=-3 --long=25 --single-thread}"
 : "${PRUNE_BACKUPS_DAYS:=7}"
+  "${PRUNE_BACKUPS_COUNT:=}"
```

Comment:
Ah, missing the colon in the start of the line. It's an obscure bash syntax to perform a no-op.